Python/Batch/common/helpers.py

```diff
@@ -476,14 +478,11 @@ def download_blob_from_container(block_blob_client,
 
     destination_file_path = os.path.join(directory_path, blob_name)
 
-    block_blob_client.get_blob_to_path(container_name,
-                                       blob_name,
-                                       destination_file_path)
+    block_blob_client.get_blob_to_path(
+        container_name, blob_name, destination_file_path)
 
     print('  Downloaded blob [{}] from container [{}] to {}'.format(
-         blob_name,
-         container_name,
-         destination_file_path))
+        blob_name, container_name, destination_file_path))
```

Comment:
This change seems to be stylistic.

I assume the new way is the "pythonic" way to do things? If so, awesome.
I'm asking because I happen to prefer the way it was =)